docs/dev_guide/diagnostic_ref/dpct1050.rst

```diff
@@ -24,4 +24,129 @@ in such cases.
 Suggestions to Fix
 ------------------
 
-Replace the "dpct_placeholder" with the real argument.
\ No newline at end of file
+Replace the "dpct_placeholder" with the real argument.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+    __global__ void kernel(const cudaTextureObject_t texObj) {}
+    
+    void foo() {
+      float4 *d_data42;
+      cudaArray_t a42;
+      cudaMalloc(&d_data42, sizeof(float4) * 32 * 32);
+      cudaChannelFormatDesc desc42 = cudaCreateChannelDesc(32, 32, 32, 32, cudaChannelFormatKindFloat);
+      cudaMallocArray(&a42, &desc42, 32, 32);
+      cudaMemcpyToArray(a42, 0, 0, d_data42, 32 * 32 * sizeof(float4), cudaMemcpyDeviceToDevice);
+      cudaTextureObject_t tex42;
+      cudaResourceDesc res42;
+      cudaTextureDesc texDesc42;
+      res42.resType = cudaResourceTypeArray;
+      res42.res.array.array = a42;
+      cudaCreateTextureObject(&tex42, &res42, &texDesc42, NULL);
+      kernel<<<1, 1>>>(tex42);
+    }
+
+results in the following migrated SYCL code:

```

Comment:
Add "*" to SYCL - not yet attributed on page. Need to attribute first use only.

Reply:
done

Reply:
done